manifests/init.pp

```diff
@@ -31,6 +31,7 @@
   $domain_change                                        = false,
   $sitenameoverride                                     = 'UNSET',
   $vas_conf_client_addrs                                = 'UNSET',
+  $vas_conf_vasypd_update_interval                      = 'USE_DEFAULTS',
```

Comment:
USE_DEFAULTS is mostly used to be able to specify different default values depending on other details.
I don't see this here, so please just set the default to the original value of '1800'. A simple integer validation of $vas_conf_vasypd_update_interval is good enough afterwards.